opossum-tool/OpossumUI
# feat: eliminate List duplication

### Summary of changes

- merge the two List components into a single component
- delete redundant types

### Context and reason for change

Remove unnecessary code duplication.

### How can the changes be tested

Check that all places where virtualized lists are used still look and work correctly. Mostly this affects the resource browser, attribution view and package list.
